src/bin/mosaic-start.ts

```diff
@@ -132,6 +133,15 @@ mosaic
 
         await (new Graph(graphDescription).start());
       }
+      // printing of endpoints on console.
+      const endPointsTable = markdownTable([
+        ['', 'rpc port', 'ws port'],
+        [chain, rpcPort, websocketPort],
+        ['Graph node', graphDescription.rpcPort, graphDescription.websocketPort],
```

Comment:
also there are more ports we would need as inputs (`graphAdminRPC /  graphIPFS`) for commands like `subgrph`. Please look through the code and print all such values for both chain & graph nodes